src/test/handlers/featureFlags.test.js

```diff
@@ -0,0 +1,59 @@
+const request = require('supertest');
+const config = require('../../config/config.js');
+const support = require('./_support.js');
+
+describe('GET /featureflags', () => {
+    describe('Failures:', () => {
+        test('should return 200 and false for transferVerification when config is set to "false"', async () => {
+            const confOverride = JSON.parse(JSON.stringify(config));
+            confOverride.featureFlags.transferVerification = false;
+
+            const db = support.createDb();
+            const server = support.createServer(db, undefined, undefined, confOverride);
+
+            const response = await request(server).get('/featureflags');
+            expect(response.status).toEqual(200);
+            expect(response.body).toEqual({
+                transferVerification: false,
+            });
+
+            server.close();
+        });
+
+        test('should return 200 and true for transferVerification when config is set to "true"', async () => {
+            const confOverride = JSON.parse(JSON.stringify(config));
+            confOverride.featureFlags.transferVerification = true;
+
+            const db = support.createDb();
+            const server = support.createServer(db, undefined, undefined, confOverride);
+
+            const response = await request(server).get('/featureflags');
+            expect(response.status).toEqual(200);
+            expect(response.body).toEqual({
+                transferVerification: true,
+            });
+
+            server.close();
+        });
+
+        test('should return 200 and values for any additional feature flags when set in config"', async () => {
+            const confOverride = JSON.parse(JSON.stringify(config));
+            confOverride.featureFlags.transferVerification = true;
```

Comment:
I would leverage the beforeEach block in order to reset the global objects that the tests have altered like this one, otherwise you have to keep resetting them in every test and that is prone to causing bugs in future development where you don't have to check all the detailed implementation of each test to figure out why a value isn't what you expected.